components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -97,6 +143,13 @@ service NodesService {
 			action: "infra:nodes:list"
 		};
 	};
+
+	/*
+	Rerun a node
```

Comment:
Does this only determine if a node is reachable?

Reply:
yes